orderer/consensus/smartbft/chain_test.go

```diff
@@ -358,6 +359,9 @@ func TestAddAndRemoveNodeWithoutStop(t *testing.T) {
 	dir := t.TempDir()
 	channelId := "testchannel"
 
+	flogging.ActivateSpec("debug")
```

Comment:
why do we need this?

Reply:
The default logs seem to be Info.
In case of failure I would like to see debug logs from the smartbft library

Reply:
Why? the tests aren't flaky in CI

Reply:
https://github.com/hyperledger/fabric/blob/main/orderer/consensus/smartbft/synchronizer_bft_test.go#L43

like what's done here
and there's more than one place like this.

Reply:
Right. The TestAddAndRemoveNodeWithoutStop test is not stable. Quite often it terminates with an error. I can't catch the error and fix it yet.